client/src/index.css

```diff
@@ -16,11 +16,9 @@ body {
   /* FAQ: Use variable to isolate value from application of value */
   /* WARNING: Must use a unit, NOT a keyword (which would break any `calc()`) */
   /* stylelint-disable length-zero-no-unit */
-  --min-height: 0px;
-  --min-width: 0px;
+  min-height: var(--min-height, 0);
+  min-width: var(--min-width, 0);
   /* stylelint-enable length-zero-no-unit */
-  min-height: var(--min-height);
-  min-width: var(--min-width);
 }
```

Comment:
__Explanation__

This:
```
--min-height: 0;
min-height: var(--min-height);
```

Equals:
```
min-height: var(--min-height, 0);
```

Reference: [MDN: var(): "… fallbacks …"](https://developer.mozilla.org/en-US/docs/Web/CSS/var()#custom_properties_with_fallbacks_for_use_when_the_property_has_not_been_set)

---

P.S. I only just recently began to put this in practice—tested on CMS header redesign (https://github.com/TACC/Core-CMS/pull/222) a lot. _I had forgotten this feature when initially implementing this code on Portal._